### PR TITLE
HHH-15577 Metamodel Generator documentation links to 404

### PR DESCRIPTION
**Pull request by misselvexu**
https://hibernate.atlassian.net/browse/HHH-15577

### CODE REVIEW

**Comment by gsmet** [comment]
@dreab8 @beikov can one of you look at that PR?

File: orm/tooling.adoc [hunk 1]
```diff
@@ -68,7 +68,7 @@ To use Hibernate Metamodel Generator add the following dependency to your build
 </dependency>
 ----
 
-For more information and setup options refer to the link:https://docs.jboss.org/hibernate/orm/{version_family}/topical/html_single/metamodelgen/MetamodelGenerator.html[online documentation].
+For more information and setup options refer to the link:https://docs.jboss.org/hibernate/orm/{version_family}/userguide/html_single/Hibernate_User_Guide.html#tooling[online documentation].
```
Comment (dreab8) [inline orm/tooling.adoc]: probably the link should be https://docs.jboss.org/hibernate/orm/{version_family}/userguide/html_single/Hibernate_User_Guide.html#tooling-modelgen

**Comment by gsmet** [comment]
@beikov I see the PR was against the staging environment. It probably needs a cherry-pick to the production one.

**Comment by beikov** [comment]
Thanks for the reminder. Almost forgot to do that.

**Pull request merged**